src/build.ts

```diff
@@ -3,24 +3,30 @@
 import { IOType } from 'child_process';
 import { join } from 'path';
 
+import { runCommand as runStableBuildCommand } from '#commands/build/index';
 import { runCommand as runCleanCommand } from '#commands/clean';
-import { runCommand as runStableCompileCommand } from '#commands/compile/index';
-import { runCommand as runGenerateTypesCommand } from '#commands/generate_types/index';
-import { runCommand as runInstallDfxExtensionCommand } from '#commands/install_dfx_extension';
-import { runCommand as runInstallGlobalDependenciesCommand } from '#commands/install_global_dependencies/index';
+import { runCommand as runDevSetupCommand } from '#commands/dev/setup/index';
+import { runCommand as runDevTemplateCommand } from '#commands/dev/template';
+import { runCommand as runExtensionInstallCommand } from '#commands/extension/install';
+import { runCommand as runGenerateCommand } from '#commands/generate/index';
 import { runCommand as runNewCommand } from '#commands/new';
-import { runCommand as runStableTemplateCommand } from '#commands/template';
 import { runCommand as runVersionCommand } from '#commands/version';
-import { runCommand as runExperimentalCompileCommand } from '#experimental/commands/compile/index';
-import { runCommand as runExperimentalTemplateCommand } from '#experimental/commands/template';
-import { runCommand as runUploadAssetsCommand } from '#experimental/commands/upload_assets/index';
+import { runCommand as runExperimentalBuildCommand } from '#experimental/commands/build/index';
+import { runCommand as runExperimentalTemplateCommand } from '#experimental/commands/dev/template';
+import { runCommand as runExperimentalUploadAssetsCommand } from '#experimental/commands/upload_assets/index';
 import {
     experimentalMessageCli,
     experimentalMessageDfxJson
 } from '#experimental/utils/experimental_message';
+import { Command as ExperimentalCommand } from '#experimental/utils/types';
 import { getCanisterConfig } from '#utils/get_canister_config';
-import { AZLE_PACKAGE_PATH } from '#utils/global_paths';
-import { CanisterConfig, Command } from '#utils/types';
+import { AZLE_ROOT } from '#utils/global_paths';
+import {
+    CanisterConfig,
+    Command as StableCommand,
```

Comment:
I think everywhere, stable things generally speaking, should not say Stable in their names...besides maybe the filepath itself. So `StableCommand` should just be `Command`, and if there is an experimental version it would be called `ExperimentalCommand`.

Reply:
done